runtime/statemine/src/lib.rs

```diff
@@ -273,11 +274,12 @@ parameter_types! {
 }
 
 impl pallet_authorship::Config for Runtime {
-	// type FindAuthor = pallet_collator_selection::FindAuthorFromIndex<Self, Aura>;
+	// TODO https://github.com/paritytech/statemint/issues/23
+	// Add FindAccountFromAuthorIndex when Aura is integrated
 	type FindAuthor = ();
 	type UncleGenerations = UncleGenerations;
 	type FilterUncle = ();
-	type EventHandler = (CollatorSelection,);
```

Comment:
removing this will break the rewards and boot mechanism of collator selection, any reason here?

Reply:
No, I got it from Statemint runtime. It wasn't there but Kian just added it on a following PR.